cpython/Lib/concurrent/futures/__init__.py

```diff
@@ -14,5 +14,11 @@
                                       Executor,
                                       wait,
                                       as_completed)
-from concurrent.futures.process import ProcessPoolExecutor
+try:
+    from concurrent.futures.process import ProcessPoolExecutor
+except ImportError:
+    import sys
+    if sys.platform != 'uwp':
```

Comment:
Would it make more sense to do this check _before_ attempting to import the module?